packages/@react-aria/selection/src/ListKeyboardDelegate.ts

```diff
@@ -93,14 +130,23 @@ export class ListKeyboardDelegate<T> implements KeyboardDelegate {
       return null;
     }
 
-    let pageY = Math.max(0, item.offsetTop + item.offsetHeight - menu.offsetHeight);
+    if (this.orientation === 'horizontal') {
+      let pageX = Math.max(0, item.offsetLeft + item.offsetWidth - menu.offsetWidth);
+
+      while (item && item.offsetLeft > pageX) {
+        key = this.getKeyAbove(key);
+        item = key == null ? null : this.getItem(key);
```

Comment:
Yeah because in TagGroup there is no horizontal scrolling. So it just goes as far as it can.

Reply:
By "as far as it can go" does that mean PageDown should go to the last tag then? This [story](https://reactspectrum.blob.core.windows.net/reactspectrum/5a10dc0f750e872582d5ad37ac1deec60ff416c1/storybook/index.html?path=/story/taggroup--default&providerSwitcher-express=false) only moves to the second tag when PageDown is pressed, and in each of the other stories focus either [doesn't move](https://reactspectrum.blob.core.windows.net/reactspectrum/5a10dc0f750e872582d5ad37ac1deec60ff416c1/storybook/index.html?path=/story/taggroup--with-icons&providerSwitcher-express=false) or goes to the [third tag](https://reactspectrum.blob.core.windows.net/reactspectrum/5a10dc0f750e872582d5ad37ac1deec60ff416c1/storybook/index.html?path=/story/taggroup--on-remove&providerSwitcher-express=false)

Reply:
oh that's weird. lemme look into it

Reply:
hmmm... I don't actually see PageDown doing anything, PageUp works fine but nothing happens if I press PageDown, no matter what tag I start on

Reply:
I think it depends on your window width haha. It's because TagGroup is not actually scrollable, so `offsetLeft` is relative to the whole page. I think I'll just make it go to the first/last item if the `offsetParent !== menu` here.